xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/test/java/org/xwiki/rendering/macro/descriptor/DefaultMacroDescriptorTest.java

```diff
@@ -244,4 +250,17 @@ public void testParameterDescriptorWithDescriptionAndMandatoryOnSetter()
         Assert.assertSame(boolean.class, param3Descriptor.getParameterType());
         Assert.assertEquals(true, param3Descriptor.isMandatory());
     }
+
+    @Test
+    public void testLegacyParameterDescriptor()
+    {
+        Map<String, ParameterDescriptor> map = this.macroDescriptor.getParameterDescriptorMap();
+
+        ParameterDescriptor deprecatedDescriptor = new LegacyParameterDescriptor(map.get("deprecatedparameter"));
+        Assert.assertFalse(deprecatedDescriptor.isDeprecated());
+
+        ParameterDescriptor advancedDescriptor = new LegacyParameterDescriptor(map.get("advancedparameter"));
+        Assert.assertFalse(advancedDescriptor.isAdvanced());
+        Assert.assertEquals(Collections.emptyList(), advancedDescriptor.getGroup());
```

Comment:
Please use a static import for Assert as much as possible.

Reply:
I will also use JUnit5 along with one test method and simplify some assertions.